<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeatConnection</class>
 <widget class="QWidget" name="NeatConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="ConnectionCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ConnectionLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ConnectionComboBoxIn"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ConnectionComboBoxOut"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Weight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ConnectionDoubleSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000001000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ConnectionCheckBoxEnable">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
